File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyTreeViewElementsRow.cpp

#include "SelectRowCommand_ViewMyTreeViewElementsRow.h"
#include "SelectRowCommand_ViewMyTreeViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewMyTreeViewElementsRow::getColumnCellLabelText( ) 
  {
    return this->columnCellLabelText;
  }
  int SelectRowCommand_ViewMyTreeViewElementsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
  int SelectRowCommand_ViewMyTreeViewElementsRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyTreeViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewMyTreeViewElementsRow.cs
/// </filename>


namespace commands
{
    internal class SelectRowCommand_ViewMyTreeViewElementsRow
  {
        private string columnCellLabelText;
     
        public     virtual string getColumnCellLabelText()
    {
      return this.columnCellLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

